scss/tokens/_raw.scss

```diff
@@ -16,4 +16,35 @@ $ouds-opacity-500: .32 !default;
 $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .80 !default;
 $ouds-opacity-900: 1 !default;
-// scss-docs-end ouds-opacities-variables
+// scss-docs-end ouds-raw-opacity
+
+// Border
+// scss-docs-start ouds-raw-border
+$ouds-border-base:      4px !default;
+
+// $ouds-border-width-0:   0 !default;
+$ouds-border-width-25:  $ouds-border-base * .25 !default;
+$ouds-border-width-50:  $ouds-border-base * .5 !default;
+$ouds-border-width-75:  $ouds-border-base * .75 !default;
+$ouds-border-width-100: $ouds-border-base !default;
+// $ouds-border-width-150: $ouds-border-base * 1.5 !default;
+// $ouds-border-width-200: $ouds-border-base * 2 !default;
+
+$ouds-border-radius-0:    0 !default;
+// $ouds-border-radius-25:   $ouds-border-base * .25 !default;
+// $ouds-border-radius-50:   $ouds-border-base * .5 !default;
+$ouds-border-radius-75:   $ouds-border-base * .75 !default;
+// $ouds-border-radius-100:  $ouds-border-base !default;
+$ouds-border-radius-150:  $ouds-border-base * 1.5 !default;
+// $ouds-border-radius-200:  $ouds-border-base * 2 !default;
+$ouds-border-radius-300:  $ouds-border-base * 3 !default;
+// $ouds-border-radius-400:  $ouds-border-base * 4 !default;
+// $ouds-border-radius-500:  $ouds-border-base * 5 !default;
+// $ouds-border-radius-600:  $ouds-border-base * 6 !default;
+// $ouds-border-radius-800:  $ouds-border-base * 8 !default;
+
```

Comment:
I don't remember precisely what we said, but there's still a design token like this in our references outside of this PR. Let's check if we need to update the reference of this PR:
* `border-radius-9999` → `2000`

Reply:
OK, decision taken in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2687#discussion_r1740871202. I resolve this comment.